packages/SwingSet/src/kernel/vatTranslator.js

```diff
@@ -375,34 +476,57 @@ function makeTranslateVatSyscallToKernelSyscall(vatID, kernelKeeper) {
    * @returns { KernelSyscallObject }
    */
   function vatSyscallToKernelSyscall(vsc) {
-    const [type, ...args] = vsc;
-    switch (type) {
-      case 'send':
+    switch (vsc[0]) {
+      case 'send': {
```

Comment:
See prior comment about uglification in `kernelDeliveryToVatDelivery`.  Same thing here.